sessionctx/variable/session.go

```diff
@@ -548,6 +548,7 @@ func NewSessionVars() *SessionVars {
 		enableStreaming = "0"
 	}
 	terror.Log(vars.SetSystemVar(TiDBEnableStreaming, enableStreaming))
+	terror.Log(vars.SetSystemVar(TIDBMemQuotaQuery, strconv.FormatInt(vars.MemQuota.MemQuotaQuery, 10)))
```

Comment:
Does this bug also affect other variables?

Reply:
In tidb 3.0, only support hot reload some config.
```
var hotReloadConfigItems = []string{"Performance.MaxProcs", "Performance.MaxMemory", "Performance.CrossJoin",
	"Performance.FeedbackProbability", "Performance.QueryFeedbackLimit", "Performance.PseudoEstimateRatio",
	"OOMAction", "MemQuotaQuery", "StmtSummary.MaxStmtCount", "StmtSummary.MaxSQLLength", "TiKVClient.StoreLimit"}
```
In this list, only `MemQuotaQuery` and `TiKVClient.StoreLimit` has defined variable.
`MemQuotaQuery`'s scope is Session, but not init session variable when creating session.
`TiKVClient.StoreLimit`'s scope is Global/Session, so it get global variable and set session variable when creating session.

So that, only MemQuotaQuery will cause the bug.